release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -232,6 +238,14 @@ class RepoInfoExpansionState extends State<RepoInfoExpansion> {
     });
   }
 
+  /// Helper function to determine if a [UrlButton] should be rendered instead of a [SelectableText].
+  bool isUrlButton(String repoElement) {
```

Comment:
`isClickable` would be more accurate.

Would it be better to generalize this and check if the value starts with http?